.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+main
```

Comment:
what dis?

Reply:
There was a compiled binary I(?) Accidentally committed sometime, I deleted it and added it to .gitignore for good measure

Reply:
Comment for future reference in the .gitignore would be good (totally makes sense to add the binary to be ignored)

Reply:
:+1: